test/gst-vaapi/vpp/deinterlace.py

```diff
@@ -6,16 +6,13 @@
 
 from ....lib import *
 from ..util import *
+from .vpp import VppTest
 
 if len(load_test_spec("vpp", "deinterlace")):
   slash.logger.warn(
     "gst-vaapi: vpp deinterlace with raw input is no longer supported")
 
-@slash.requires(have_gst)
-@slash.requires(*have_gst_element("vaapi"))
-@slash.requires(*have_gst_element("vaapipostproc"))
-@slash.requires(*have_gst_element("checksumsink2"))
-class DeinterlaceTest(slash.Test):
+class default(VppTest):
```

Comment:
This should stay as `DeinterlaceTest` since it's only a base class

Reply:
will fix it